cmd/osbuild-worker/jobimpl-osbuild-koji.go

```diff
@@ -86,6 +86,28 @@ func (impl *OSBuildKojiJobImpl) Run(job worker.Job) error {
 		return err
 	}
 
+	// In case the manifest is empty, try to get it from dynamic args
+	if len(args.Manifest) == 0 {
+		if job.NDynamicArgs() > 1 {
+			var manifestJR worker.ManifestJobByIDResult
+			err = job.DynamicArgs(1, &manifestJR)
+			if err != nil {
+				return err
+			}
+
+			// skip the job if the manifest generation failed
+			if manifestJR.Error != "" {
```

Comment:
You will probably have to check for `JobErrors` now since https://github.com/osbuild/osbuild-composer/pull/2192 was merged

Reply:
Rebased on this. Please have a look though :)

Reply:
I think you got them all except this one :p

Reply:
LOL. oops.

Reply:
Done ;)